### PR TITLE
Bug 1137796: messages for users uploading files to tooltool

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: tooltool.py [hunk 1]
```diff
@@ -755,6 +756,24 @@ def upload(package, user, host, path):
     cmd2 = "rsync  %s/*.txt %s@%s:%s --progress" % (package, user, host, path)
     cmd3 = "rsync  %s/*.tt %s@%s:%s --progress" % (package, user, host, path)
 
+    print textwrap.dedent("""\
+        **SECURITY WARNING**
+
+        Uploading files to tooltool is a risky operation.  Please consider:
+
+          * Is this data PUBLIC or INTERNAL?  If not don't upload it!
```
Comment (gdestuynder) [inline tooltool.py]: If **not do not upload it!** (emphasis)

File: tooltool.py [hunk 2]
```diff
@@ -755,6 +756,24 @@ def upload(package, user, host, path):
...
+            INTERNAL data is non-public data that is not secret, e.g., non-redistributable binaries.
+            See https://mana.mozilla.org/wiki/display/ITSECURITY/IT+Data+security+guidelines
+            INTERNAL data should not be uploaded to the `pub` directory.
```
Comment (gdestuynder) [inline tooltool.py]: should **not** (emphasis)

File: tooltool.py [hunk 3]
```diff
@@ -755,6 +756,24 @@ def upload(package, user, host, path):
     cmd2 = "rsync  %s/*.txt %s@%s:%s --progress" % (package, user, host, path)
     cmd3 = "rsync  %s/*.tt %s@%s:%s --progress" % (package, user, host, path)
 
+    print textwrap.dedent("""\
+        **SECURITY WARNING**
+
+        Uploading files to tooltool is a risky operation.  Please consider:
+
+          * Is this data PUBLIC or INTERNAL?  If not don't upload it!
+            INTERNAL data is non-public data that is not secret, e.g., non-redistributable binaries.
```
Comment (gdestuynder) [inline tooltool.py]: "data that is not intended for public distribution, but that is **not sensitive**, e.g. non-redistributable binaries" (maybe clearer)

**Comment by Callek** [comment]
You weren't the first, but can you push `master` to https://git.mozilla.org/?p=build/tooltool.git since that is RoR for tooltool.

(I'd argue we should swap RoR, but since thats not the case...)

**Comment by djmitche** [comment]
done

**Pull request merged**